hashes/aws-lc

```diff
@@ -1,2 +1,8 @@
-# https://github.com/aws/aws-lc/archive/refs/tags/AWS-LC-FIPS-2.0.17.tar.gz
-SHA512 (AWS-LC-FIPS-2.0.17.tar.gz) = acb61f71d4039112ab027db56a9b5b1f9b272d41bd014cb733a71f365db13a2ade4ecf86d988900b2f7c8429086caf379a59803bae3c7a00111da0b72b357a2a
+# https://github.com/aws/aws-lc/archive/AWS-LC-FIPS-3.0.0/aws-lc-AWS-LC-FIPS-3.0.0.tar.gz
+### Additional patches from AWS-LC-FIPS 3.0 branch:
+### git clone https://github.com/aws/aws-lc
+### cd aws-lc
+### git checkout origin/fips-2024-09-27
+### git format-patch --start-number=1001 AWS-LC-FIPS-3.0.0..
```

Comment:
Please bring back `--no-numbered` to avoid embedding the sequence numbers in the patch files. Otherwise they all get modified whenever a new patch is added.

Reply:
In that case, do we want to rename the [0001-Avoid-weak-symbols-when-building-for-Go.patch](https://github.com/bottlerocket-os/bottlerocket-sdk/pull/266/files#diff-192be1cf758a4f3f7dddc76949699a4ebf654e92d71f749f2c6e3e77e76701c2) with different prefix? 

Otherwise, there will be two patch files with same 0001 prefix.

Reply:
Just updated patch with --no-numbered. Full command
```
git format-patch --start-number=1001 --no-numbered
 AWS-LC-FIPS-3.0.0..
```